arm-iothub/2016-02-03/swagger/iothub.json

```diff
@@ -1115,91 +1146,88 @@
+      ]
     },
     "EventHubProperties": {
-      "description": "The properties of the provisioned event hub used by the Iot Hub.",
+      "description": "The properties of the provisioned Event Hub-compatible endpoint used by the IoT hub.",
       "type": "object",
       "properties": {
         "retentionTimeInDays": {
           "format": "int64",
-          "description": "The retention time in days. Range of values [For F1: 1-1, S1: 1-7, S2: 1-7, S3: 1-7].",
+          "description": "The retention time for device-to-cloud messages in days. See: https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging#device-to-cloud-messages",
           "type": "integer"
         },
         "partitionCount": {
           "format": "int32",
-          "description": "The partition count. Range of values [For F1: 2-2, S1: 2-128, S2: 2-128, S3: 2-128].",
+          "description": "The number of paritions for receiving device-to-cloud messages in the Event Hub-compatible endpoint. See: https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-messaging#device-to-cloud-messages.",
           "type": "integer"
         },
         "partitionIds": {
-          "description": "The partition ids.",
+          "description": "The partition ids in the Event Hub-compatible endpoint.",
+          "readOnly": true,
           "type": "array",
           "items": {
             "type": "string"
           }
         },
         "path": {
-          "description": "The eventhub path.",
-          "type": "string"
+          "description": "The Event Hub-compatible name.",
+          "type": "string",
+          "readOnly": true
         },
         "endpoint": {
-          "description": "The endpoint.",
-          "type": "string"
-        },
-        "internalAuthorizationPolicies": {
-          "description": "The internal authorization rules.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/SharedAccessAuthorizationRule"
-          }
-        },
-        "authorizationPolicies": {
-          "description": "The authorization rules.",
-          "type": "array",
-          "items": {
-            "$ref": "#/definitions/SharedAccessAuthorizationRule"
-          }
+          "description": "The Event Hub-compatible endpoint.",
+          "type": "string",
+          "readOnly": true
         }
       }
     },
     "StorageEndpointProperties": {
-      "description": "The properties of the Storage Endpoint for file upload.",
+      "description": "The properties of the Azure Storage endpoint for file upload.",
       "type": "object",
       "properties": {
         "sasTtlAsIso8601": {
-          "description": "SAS time to live. Range: 1 Min (PT1M) - 1 Day (P1D).",
+          "description": "The period of time for which the the SAS URI generated by IoT Hub for file upload is valid. See: https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-file-upload#file-upload-notification-configuration-options.",
```

Comment:
The doc url in the description says that the default is 1 hour. Please update the default value in the swagger spec.